### ISSUE TITLE
Postman 3 does not start

### ISSUE THREAD

**Issue by cnicodeme** [a85/POSTMan-Chrome-Extension#1564]
Hi,

I'm trying to start Postman 3. Chrome tells me the app is loaded, I can find it in the processes from Chrome and from my OS, but no new window is shown, no changes, it's like Postman is not displayed at all.

I'm using Fedora 22 with Gnome 3 in a 64bit environment.

I'd like to give you more details if you tell me how (Chrome debug ? stack trace ?)

**Comment by abhijitkane** [comment]
Can you enable debugging for packed apps (http://blog.getpostman.com/2015/06/13/debugging-postman-requests/) and restart the browser? Once this is done, open Postman and head to chrome://inspect/#apps and check if you see a listing for Postman. If you do, hit 'Inspect' and check for any errors in the Console.

**Comment by cnicodeme** [comment]
Yes ! great, there is issues in the console

There is three issues : 

```
extensions::platformApp:17 unload is not available in packaged apps.

http://fhbjgbiflinjbdggehcddcbncdddomop/socket.io/1/?__sails_io_sdk_version…io_sdk_platform=browser&__sails_io_sdk_language=javascript&t=1447410483403 Failed to load resource: net::ERR_NAME_NOT_RESOLVED

extensions::platformApp:17 beforeunload is not available in packaged apps.
```

If you need more details, please ask what and how, I'll be happy to help ! :)

Best,Cyril

**Comment by cnicodeme** [comment]
(Note, when I expand the errors indicated previously, I don't have any stack trace or more details, only : 
"(anonymous function) @ @extensions::platformApp:17" for example.

**Comment by abhijitkane** [comment]
That's odd..those are expected errors. There are a number of users running Postman on linux. Can you try reinstalling Postman? (Uninstall from chrome://extensions)

**Comment by cnicodeme** [comment]
I already did multiple times without any luck.

Postman is launching correctly, it's shown in the process toolbar (at the top, in Gnome), when I click on it, I have "Quit" option, but the window never appears.

It's like it would be displayed outside my screens : it's launched, but not visible.

If that may help, I have a dual screen configuration. I switched the settings to one screen enabled in case it would fixe the issue, but without any luck.

**Comment by abhijitkane** [comment]
Is there a dock icon for Postman that shows up? If so, you can try right clicking on it and minimize/maximize the window. This has worked for a few Windows users.

**Comment by cnicodeme** [comment]
There is a dock icon, but the only options are it's name (make it in front), or to quit. No resize window.

But you made me think of something. Sometimes, chrome starts new windows in odd sizes. So I opened Postman (which was not displayed), and hit the "Maximize" keyboard shortcut.

It worked !

Then I resized it to small, only to find out it was starting with a width of 1px ... That was the reason !

Thanks for your help. I can close the ticket now!
